yb-voyager/src/cp/control_plane.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cp
+
+import (
+	"github.com/google/uuid"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+type ControlPlane interface {
+	Init() error
+	ExportSchemaStarted(*ExportSchemaEvent)
+	ExportSchemaCompleted(*ExportSchemaEvent) // Only success is reported.
```

Comment:
For consistency, have two separate types for the two `Event`s. Even though today their internal contents are the same, in future, the contents of the events can differ. So better to have a separate type for each separate event. As suggested in one of the later comments, while defining these event types, you can factor out common types in `BaseEvent` and embed it in  the specific Event types.

Also name type event types by appending `Event` to the event name e.g.
```
ExportSchemaStarted(*ExportSchemaStartedEvent)
ExportSchemaCompleted(*ExportSchemaCompletedEvent)
```
Follow the naming pattern of:
```
SomeActionPerformed(*SomeActionPerformedEvent)
```
Fix all such occurrences.

Reply:
Incoorporated the comments.